Documentation/Migration/Index.rst

```diff
@@ -179,11 +174,20 @@ PHP-based rendering tool:
     rendering, and every file that is just a code snippet placed in an external file
     should be renamed to use a :file:`.rst.txt` extension instead.
 
-Recommendations
-===============
+
+Render your Documentation files locally
+=======================================
+
+Use our Docker container to render your documentation locally. Read more about it in
```

Comment:
I think I would like to keep this section at the top? Otherwise we might lose people thinking it's too hard to get this started, if they see a large block of text?

Reply:
I agree

Reply:
I chatted with @Alagts  about it and we came to an agreement to change that section to something like:

```
While you could already jump in and render your docs directly (see :ref:), we suggest to clean up some other things first:
```

This could then stand where the current heading stands, and link to the details... that would maybe make both sides happy, those who want to start rendering as soon as its possible, and those would also like to earn some cherries on top for finishing everything and rendering, when all is properly done.

Reply:
Sounds good